test/e2e/testnet/node.go

```diff
@@ -49,14 +49,14 @@ type Node struct {
 	traceProxyHost string
 }
 
-// PullRoundStateTraces retrieves the round state traces from a node.
+// PullBlockSummaryTraces retrieves the block summary traces from a node.
 // It will save them to the provided path.
-func (n *Node) PullRoundStateTraces(path string) ([]trace.Event[schema.RoundState], error,
+func (n *Node) PullBlockSummaryTraces(path string) ([]trace.Event[schema.BlockSummary], error,
```

Comment:
do we care that this is technically breaking? (test! instead of test)

Reply:
Good point, I retracted the old `PullRoundStateTraces` func to make this PR non-breaking.